### ISSUE TITLE
A11y: Docs examples -- duplicate IDs

### ISSUE THREAD

**Issue by joasmi723** [cengage/react-magma#273]
There are a few a11y issues with some of the examples in the docs.  These should be quick fixes just to the docs pages, not to the components themselves.

1) We have a few pages in the docs that have some duplicate IDs, so are throwing a11y errors. 
Pages include:
-- Combobox
-- Radio buttons
-- Select
-- Nav Tabs
-- Search

2)  On the Tables example we have some inputs without labels.  These should be given (visually hidden) labels.

3) On the Cards page, we have an image without an alt attribute.

4) Some examples on the IconButtons page need adjusting. Under the sections sizes, shapes, and full-width there are a few examples that have the aria-label placed on the icon instead of on the button itself.

**Comment by joasmi723** [comment]
We also have some examples on the IconButtons page that need adjusting.  Under the sections sizes, shapes, and full-width there are a few examples that have the `aria-label` placed on the icon instead of on the button itself.  These examples should be updated.

**Comment by silvalaura** [comment]
@Uditsingh67 
Both of these things are expected and not a bug.
1. NavTabs is just an example so it's okay that the contrast is not right.
2. We expect the disabled state to not have sufficient contrast. 

The a11y issues mentioned above are for the documentation site, not for storybook.

**Comment by Uditsingh67** [comment]
Hi @silvalaura 
Can you please share URLs for :
 Pages include:
-- Combobox
-- Radio buttons
-- Select
-- Nav Tabs
-- Search

**Comment by silvalaura** [comment]
> Hi @silvalaura Can you please share URLs for : Pages include: -- Combobox -- Radio buttons -- Select -- Nav Tabs -- Search

All those pages can be found in our documentation site: https://next--upbeat-sinoussi-f675aa.netlify.app/ 
-- Combobox: https://next--upbeat-sinoussi-f675aa.netlify.app/api/combobox/
-- Radio buttons: https://next--upbeat-sinoussi-f675aa.netlify.app/api/radio/
-- Select: https://next--upbeat-sinoussi-f675aa.netlify.app/api/select/
-- Nav Tabs: https://next--upbeat-sinoussi-f675aa.netlify.app/api/nav-tabs/
-- Search: https://next--upbeat-sinoussi-f675aa.netlify.app/api/search/

**Comment by Uditsingh67** [comment]
The issue is fixed:
Pages include:
-- Combobox **: PASS**
-- Radio buttons **: PASS**
-- Select **: PASS**
-- Nav Tabs **: PASS**
-- Search **: PASS**
